config-vm.md

```diff
@@ -0,0 +1,37 @@
+# Virtual-machine-specific Configuration
+
+Virtual-machine-based runtimes require additional configuration to that specified in the [default spec configuration](config.md).
+
+This optional configuration is specified in a "VM" object:
+
+* **`imagePath`** (string, required) host path to file that represents the root filesystem for the virtual machine.
```

Comment:
This might make sense as a structure, rather than just a path, to allow different kinds of images in the future.